tests/ImageSharp.Tests/RunTestsInLoop.ps1

```diff
@@ -0,0 +1,22 @@
+﻿# This script can be used to collect logs from sporadic bugs
+Param(
```

Comment:
This started as personal sandboxing, but can be useful for stressing local runs in the future, so decided to keep it.